### DISCUSSION TITLE
How do I create id = db.Column(db.Integer, unique=True) field, in postgress without serial?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [sqlalchemy/sqlalchemy#8243] [Usage Questions] [answered]
How do I create id = db.Column(db.Integer, unique=True) field, in postgress without serial?
I want just plain old INT because I have my own logic that generates IDS 
(I use sequences)

**Comment by CaselIT** [comment, marked as answer]
Hi,

That will not use serial, unless you add primary_key=True. You can also use `autoincrement` to avoid serial in primary keys
